ci-operator/step-registry/openshift-extended/test/stresstest/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
please do not change disassterrecovery owner file because I use https://github.com/openshift/release/pull/53398 to correct it.

after it is merged, please rebase the latest code.

and for your owners file, please also not use link and take real file same above comment.
```yaml
approvers:
  - jianzhangbjz
  - kuiwang02
reviewers:
  - jianzhangbjz
  - kuiwang02
  - qiliRedHat
  - xia-zhao-rh

```

Reply:
it is merged, so please rebase the latest code. and then only change your owner file to real file with above content.

please execute `make update` in your local env and then submit the files. or else, you always meet step-registry-metadata failure.